collect_app/src/main/java/org/odk/collect/android/activities/SplashScreenActivity.java

```diff
@@ -225,4 +229,9 @@ public void run() {
         };
         t.start();
     }
+
+    private void initMapBox() {
```

Comment:
I know the fix for Mapbox doesn't look good you would probably ask: can't we do the same what works for Google: https://github.com/getodk/collect/pull/3804/commits/647dc72d81aeeefc1a7f483490714950d403b31f?
Unfortunately we can't we need to display that map even if it;s not visible like in this case to make sure `getMapAsync` and then `setStyle` is called. Only then MapBox is initialized properly and we can use it offline.